### PR TITLE
add log-cb php function

### PR DESCRIPTION
**Pull request by nick-zh**
#233

### CODE REVIEW

File: rdkafka.c [hunk 1]
```diff
@@ -147,6 +147,10 @@ static void kafka_init(zval *this_ptr, rd_kafka_type_t type, zval *zconf TSRMLS_
         return;
     }
 
+    if (intern->cbs.log) {
+        rd_kafka_set_log_queue(rk, NULL);
```
Comment (arnaud-lb) [inline rdkafka.c:156]: Should we also set `log.queue=true` in the conf, before callling rd_kafka_new ? https://github.com/edenhill/librdkafka/blob/4fb7c9924c14f44c4f1490b80b8bd714098dd90b/src/rdkafka.h#L2923

Comment (nick-zh) [reply rdkafka.c:156]: @arnaud-lb yeah good input, i added to the doc that this needs to be true, but you are absolutely right, we can just set it actually, i will adjust this accordingly

Comment (nick-zh) [reply rdkafka.c:156]: @arnaud-lb done, let me know if the change is ok like that, sry for the style stuff :/ is there like a linter where i can check next time? (sry havent written c in some time)

Comment (arnaud-lb) [reply rdkafka.c:156]: Thanks :) I don't know C linters, but we should probably add one in the build at some point.

**Pull request merged**